src/worker/sandbox/cgroups/cgroup.go

```diff
@@ -53,27 +49,35 @@ func (cg *CgroupImpl) Release() {
 
 		select {
 		case cg.pool.recycled <- cg:
-			cg.printf("release and recycle")
+			if common.Conf.Trace.Cgroups {
+				cg.logger.Info("release and recycle", "CGROUP Pool", cg.pool.Name, "CGROUP", cg.name)
```

Comment:
Can't you use a logger.With call when creating the logger so we always get this info automatically?